code/src/nmea_parser.cpp

```diff
@@ -29,19 +60,19 @@ namespace r2d2::location {
                 break;
             }
             case GGA::latitude: {
-                location.latitude = atof(gps_message + i, sep_offset);
+                location.latitude = get_degrees(gps_message + i, sep_offset);
                 break;
             }
             case GGA::north_south_hemisphere: {
-                location.north_south_hemisphere = gps_message[i];
+                location.is_north_hemisphere = gps_message[i] == 'N';
```

Comment:
Why is a `char` being put in a `bool` member?

Reply:
There is no `char` in a bool. As `gps_message[i] == 'N'` returns true or false

Reply:
My  bad, I misread the statement.